src/nodes/display/FilmNode.js

```diff
@@ -0,0 +1,56 @@
+import TempNode from '../core/TempNode.js';
+import { uv } from '../accessors/UVNode.js';
+// import { luminance } from './ColorAdjustmentNode.js';
+import { addNodeElement, tslFn, nodeObject, /*vec3,*/ vec4 } from '../shadernode/ShaderNode.js';
+import { uniform } from '../core/UniformNode.js';
+import { mix, fract, clamp, rand } from '../math/MathNode.js';
+import { timerLocal } from '../utils/TimerNode.js';
+
+class FilmNode extends TempNode {
+
+	constructor( textureNode, intensity = 0.5, grayscale = false ) {
+
+		super();
+
+		this.textureNode = textureNode;
+
+		this.intensity = uniform( intensity );
+		this.grayscale = uniform( Number( grayscale ) );
```

Comment:
WebGPU does not support bool uniforms, see https://stackoverflow.com/questions/77709970/in-webgpu-what-is-the-stride-and-size-of-a-boolean.

How should we handle bool type properties parameters then? With `0|1` like demonstrated here?